src/libs/ajax/data-table-providers/WdsDataTableProvider.test.ts

```diff
@@ -644,6 +660,19 @@ describe('WdsDataTableProvider', () => {
         });
     });
   });
+
+  describe('importTdr', () => {
+    it('imports a snapshot from tdr', () => {
+      // ====== Arrange
+      const provider = new TestableWdsProvider(uuid, testProxyUrl);
+      // ====== Act
+      return provider.importTdr(uuid, uuid).then(() => {
+        // ====== Assert
+        expect(importTdr.mock.calls.length).toBe(1);
+        // expect(actual.status).toBe(202);
```

Comment:
Test disabled since refactoring of the importTdr to use `withErrorReporting` led to un `unknown` return type - not sure how to properly test it, or how to re-refactor it so it's more easily testable.